src/proto/adminctrl.proto

```diff
@@ -143,3 +144,10 @@ message GenerateTxInclusionProofRequest {
 message GenerateTxInclusionProofReply {
 	string merkle_block = 1;
 }
+
+message VerifyInclusionProofRequest {
```

Comment:
Good if we can move those methods on the `civkitservice.proto` interface. That way they can be reused by `notaryd` in the future.